docs/docs-content/clusters/edge/local-ui/cluster-management/export-cluster-definition.md

```diff
@@ -24,7 +46,115 @@ host. A cluster definition contains one or more cluster profiles, including the
 - curl is installed on your machine. You can also use API management tools such as Postman. This document uses curl as
   an example.
 
-## Export Cluster Definition
+### Instructions
+
+1. Download Palette Edge Content CLI and assign the executable bit to the CLI.
+
+   ```shell
+   VERSION=4.1.2
+   wget https://software.spectrocloud.com/stylus/v$VERSION/cli/linux/palette-edge
+   chmod +x palette-edge
+   ```
+
+2. Use the following command to move the palette-edge binary to the **/usr/local/bin** directory to make the binary
+   available in your system $PATH. This will allow you to issue the `palette-edge` command from any directory in your
+   development environment.
+
+   ```bash
+   mv palette-edge /usr/local/bin
+   ```
+
+3. Verify the installation of the Palette Edge CLI by issuing the following command. The output will display information
+   about the currently supported OS and Kubernetes distributions.
+
+   ```bash
+   palette-edge show
+   ```
+
+   ```hideClipboard bash
+   # Sample output
+   ┌────────────────────────────────────────────────────────────────────────┐
+   | OS Flavor     | Description        | Base Image URI                    |
+   | opensuse-leap | Opensuse Leap 15.4 | quay.io/kairos/core-opensuse-leap |
+   | ubuntu-20     | Ubuntu 20.4 LTS    | quay.io/kairos/core-ubuntu-20-lts |
+   | ubuntu-22     | Ubuntu 22.4 LTS    | quay.io/kairos/core-ubuntu-22-lts |
+   └────────────────────────────────────────────────────────────────────────┘
+   ┌─────────────────────────────────────────────────────────────────────────────────────────────┐
+   | K8S Flavor | Description        | Supported Versions                                        |
+   | k3s        | Rancher K3s        | 1.25.2-k3s1,1.24.6-k3s1,1.23.12-k3s1,1.22.15-k3s1         |
+   | kubeadm    | Kubernetes kubeadm | 1.25.2,1.24.6,1.23.12,1.22.15                             |
+   | rke2       | Rancher RK2        | 1.25.2-rke2r1,1.24.6-rke2r1,1.23.12-rke2r1,1.22.15-rke2r1 |
+   └─────────────────────────────────────────────────────────────────────────────────────────────┘
+   ┌─────────────────────────────────┐
+   | Component             | Version |
+   | Spectro Agent Version | v3.4.3  |
+   | Kairos Version        | v2.0.3  |
+   └─────────────────────────────────┘
+   ```
+
+4. Log in to [Palette](https://console.spectrocloud.com).
+
+5. Select the project you want to deploy the Edge host to and copy down the **Project ID**. You can find the project id
```

Comment:
You can also link to this page to help people find the project ID

https://docs.spectrocloud.com/tenant-settings/projects/#project-id